api/src/client/WebserviceLicenseStatusProcessorClient.test.ts

```diff
@@ -478,8 +478,10 @@ describe("WebserviceLicenseStatusProcessorClient", () => {
       expect(determineLicenseStatus("Withdrawn")).toBe("WITHDRAWN");
     });
 
-    it("returns UNKNOWN when status is not valid", () => {
-      expect(determineLicenseStatus("fake status")).toBe("UNKNOWN");
+    it("returns Error when status is not valid", () => {
```

Comment:
boulder - you should review how other error are throw and update test accordingly, there are other examples under webflow and dynamics. There should be test not only at the client level, but also each level the parent function is called to ensure the error is traveling back/ user gets a response.